schema/index.js

```diff
@@ -11,7 +14,26 @@ const resolvers = {};
   });
 });
 
-let rootSchema = '';
+resolvers.ObjID = new GraphQLScalarType({
+  name: 'ObjID',
+  description: 'Id representation, based on Mongo Object Ids',
+  parseValue(value) {
+    return ObjectId(value); // eslint-disable-line new-cap
```

Comment:
I sure hope we don't end up with `eslint-disable-line` in our `cgs init` output lol